src/main/scala/com/github/flakm/InMemoryLdapServer.scala

```diff
@@ -98,22 +99,29 @@ object InMemoryLdapServer {
     val ds: InMemoryDirectoryServer = new InMemoryDirectoryServer(
       settings
     )
-    object Converter {
-      import scala.jdk.CollectionConverters._
-      val allFilesFromConfiguration =
-        config.getStringList("inmemoryldap.files").asScala.toList
-    }
+    val allFilesFromConfiguration =
+      config.getStringList("inmemoryldap.files").asScala.toList
     lazy val constructPathsFromFiles: List[String] => List[String] =
       aListOfFileNames => {
         aListOfFileNames
           .map(
-            aSingleFileName => getClass().getResource(aSingleFileName).getPath
+            aSingleFileName => {
+              Try(getClass().getResource(aSingleFileName)) match {
+                case Success(anResource) => anResource.getPath
+                case Failure(exc) => {
+                  log.error(
```

Comment:
Thanks for the fixes. I still think that this fragment should be changed to the following:

```scala
throw new Exception(s"The ldiff file defined in configuration $aSingleFileName does not exist. ", exc)
```

This will result in nice exception thrown in users face and less time spent debugging nasty miss configuration.
Would you be willing to pick it up again please? :+1:

Reply:
Changed. thanks for the tip :)